### DISCUSSION TITLE
Light / Dark Version of Site Logo

### DISCUSSION THREAD

**Discussion by howellzach** [jpanther/congo#276] [Ideas]
I often use a white logo on my dark themed site. It would be great to either be able to invert the colors of this image when switching to light mode or to be able to specify a different image for light/dark.

**Comment by pitermarx** [comment]
in your params.toml file there is an appearance switcher option
![image](https://user-images.githubusercontent.com/3081298/192152029-9dabd55b-6e42-4c10-a6fe-76aa9c15640c.png)

if you set it to true there will be a button like this in the footer
![demo](https://user-images.githubusercontent.com/3081298/192152089-dda66078-3a65-4ae4-89d0-8df87b71439e.gif)

**Reply by howellzach** [reply]
Hi @pitermarx, I'm not referring to a light/dark toggle.
I mean it would be great to have an option to invert a site logo to the opposite color for the dark version of the site (or vice-versa).

For example, if you have a site currently set to "light mode", toggling could also invert the image color from black to white. I can image this either being done by literally inverting the image color or by allowing the user to specify a separate site logo for dark mode.
